tensorflow_addons/losses/python/focal_loss.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Focal loss"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import abc
+import six
+import numpy as np
+
+import tensorflow as tf
+import tensorflow.keras.backend as K
+
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.ops import nn
+from tensorflow.python.util.tf_export import keras_export
+from tensorflow.python.keras import losses
+from tensorflow_addons.utils.python import keras_utils
+
+
+@keras_export('keras.losses.SigmoidFocalCrossEntropy')
+class SigmoidFocalCrossEntropy(losses.LossFunctionWrapper):
+  """Focal loss down-weights well classified examples and focuses on the hard
```

Comment:
> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. 

Please modify the docstring here

Example:
https://github.com/tensorflow/addons/blob/ade8fc0500e4c813c31317812bd13fcacb9496e7/tensorflow_addons/losses/python/lifted.py#L110-L117

Reply:
Done